docs/src/pages/[platform]/connected-components/authenticator/customization/customization.headers-and-footers.android.mdx

```diff
@@ -0,0 +1,80 @@
+import { Image } from '@aws-amplify/ui-react';
+
+## Headers & Footers
+
+These areas in the Authenticator are the header and footer:
+
+<Image
```

Comment:
We avoid showing screenshots on the UI Docs site since they will get out of date, please remove